src/styles/blocks/header.scss

```diff
@@ -0,0 +1,21 @@
+/* #region header */
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  max-width: 100%;
+
+  padding: 0 50px;
+  border: none;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    margin-top: 10px;
```

Comment:
It's better to use shorthand for 'margin-top' and 'margin-bottom' properties to simplify the code.